MultiServer.py

```diff
@@ -792,10 +792,12 @@ def collect_player(ctx: Context, team: int, slot: int, is_group: bool = False):
                     collect_player(ctx, team, group, True)
 
 
-def get_remaining(ctx: Context, team: int, slot: int) -> typing.List[int]:
+def get_remaining(ctx: Context, team: int, slot: int, remove_hidden=False) -> typing.List[int]:
     items = []
     for location_id in ctx.locations[slot]:
         if location_id not in ctx.location_checks[team, slot]:
+            if remove_hidden and ctx.locations[slot][location_id][2] & 8 == 8:
```

Comment:
& 8 should drop all non-8 bits, so the value is 0 or 8; meaning you should be able to drop the == 8